sdk/keyvault/keyvault-certificates/src/identifier.ts

```diff
@@ -4,26 +4,16 @@
 import { parseKeyvaultIdentifier } from "./generated/utils";
 
 /**
- * Valid collection names for Key Vault Certificate identifiers.
+ * Represents a Key Vault Certificate Id and its parsed contents.
```

Comment:
Recommendation with an intention to avoid the "parsed" reference as the interface can be used independently

```suggestion
 * Represents different components of a Key Vault Certificate Id.
```

Reply:
I pushed a fix for this! Let me know what you think.